singularity/README.md

```diff
@@ -19,5 +19,10 @@ To build the container use:
 ```
 sudo singularity build /tmp/basic_cuda_conda_singularity.sif basic_cuda_conda_singularity.def
 ```
-Note, singularity containers need to be built with sudo but can be run without so you can build on your local workstation (where you have sudo) and run on a cluster (where you don't).
+
+#### Notes for PML Internal ####
```

Comment:
I think can remove this heading as this section can also apply to external users.